src/site/content/en/blog/install-thumbor/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: How to install the Thumbor image CDN
+subhead: Learn how to install the Thumbor image CDN on your own server.
+authors:
+  - katiehempenius
+date: 2019-07-29
+hero: hero.jpg
+alt: A pile of photos.
+description: |
+  Instructions on how to install Thumbor is an open-source image CDN and can be used for free to resize, compress, and transform images.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+  - images
+---
+
+Image CDNs make it easy to dynamically optimize the aesthetics and performance of your images. Unlike most image CDNs, Thumbor is  open-source and can be used for free to resize, compress, and transform images. It's suitable for production use; [Wikipedia](https://wikitech.wikimedia.org/wiki/Thumbor) and [Square](https://medium.com/square-corner-blog/dynamic-images-with-thumbor-a430a1cfcd87) both use Thumbor.
+
+This guide explains how to install Thumbor on your own server. Once installed, you’ll be able to use Thumbor as an API for transforming your images.
+
+
+## Intro
+
+You'll be installing Thumbor on a VM running Ubuntu 16.04. Ubuntu 16.04 is a very common image and these instructions are intended to work on any cloud provider. Creating a VM might sound like more work than installing Thumbor on your local machine, but the minutes that you take to create a VM will probably save you hours or days of frustration trying to get Thumbor to properly install on your local machine. Although easy to use, Thumbor is notoriously difficult to install but these instructions simplify the process. If dependencies download quickly, the installation can be completed in 5 to 10 minutes.
+
+
+## Prerequisites
+
+This post assumes that you know how to create a Ubuntu 16.04 LTS VM on a cloud platform like [Google Cloud](https://cloud.google.com/compute/docs/instances/create-start-instance), [AWS,](https://aws.amazon.com/getting-started/tutorials/launch-a-virtual-machine/) or [Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal?toc=%2Fazure%2Fvirtual-machines%2Flinux%2Ftoc.json) and how to use command line tools to set up the VM.
+
+
+## Install Thumbor Dependencies
+
+Update and upgrade Ubuntu's already-installed packages:
+
+```bash
+sudo apt-get update -y && sudo apt-get upgrade -y
+```
+
+Install pip, the package manager for Python. Later you'll install Thumbor with pip.
```

Comment:
Whenever mentioning a command line tool like `pip` style it as inline code, like this:

> Install `pip`, the package manager for Python. Later you'll install Thumbor with `pip`.

https://developers.google.com/style/command-line-terminology